Dockerfile

```diff
@@ -63,17 +63,30 @@ USER docsworker-xlarge
 
 WORKDIR ${WORK_DIRECTORY}
 
+# Get Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+RUN chmod -R 777 ${WORK_DIRECTORY}/.cargo/bin
+
+ENV PATH="${WORK_DIRECTORY}/.cargo/bin:${PATH}"
+RUN echo ${WORK_DIRECTORY}
+
 # get shared.mk
 RUN curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/makefiles/shared.mk -o shared.mk
 
 # install snooty frontend and docs-tools
 RUN git clone -b v${SNOOTY_FRONTEND_VERSION} --depth 1 https://github.com/mongodb/snooty.git       \
     && cd snooty                                                                                   \
-    && npm ci --legacy-peer-deps --omit=dev                                                        \
```

Comment:
Need to remove omit dev as the filter functionality depends on a dev dependency.

Reply:
Wondering why it's a dev dependency then?

Reply:
I had the same question! We should definitely consider moving the relevant dependencies to be installed for prod in the near future if possible